runtime/manta/src/lib.rs

```diff
@@ -65,9 +66,12 @@ use runtime_common::{
 };
 use session_key_primitives::{AuraId, NimbusId, VrfId};
 
+use xcm::latest::prelude::*;
```

Comment:
Not sure if this should pass CI, but I get compiler warning of unused import here